sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/PropertyConfigurator.java

```diff
@@ -248,14 +248,20 @@ private String getUserId() {
      */
     @SuppressWarnings("WeakerAccess")
     public void setUserId(final String userId) {
-        if (UserIdUtils.checkUserIdValidForOneCollector(userId)) {
-            Analytics.getInstance().postCommandEvenIfDisabled(new Runnable() {
-
-                @Override
-                public void run() {
-                    mUserId = UserIdUtils.getPrefixedUserId(userId);
-                }
-            });
+        if (userId == null) {
```

Comment:
why doing that since the code was returning true for null? This is more complex than before.

Reply:
Forgot to revert this, I had a commit or two in there where null was invalid

Reply:
ok